todoman/cli.py

```diff
@@ -48,6 +48,24 @@ def _validate_date_param(ctx, param, val):
         raise click.BadParameter(e)
 
 
+def _validate_start_date_param(ctx, param, val):
+    if val is None:
+        return val
+    if val.startswith('before'):
```

Comment:
Sorry, yes, this isn't why the tests fail. However, it still needs to be changed, otherwise the date parsing below will not work properly. The reason why this isn't throwing any errors is because `parse_date` is not very strict.